engine/samples/input/page.md

```diff
@@ -0,0 +1,76 @@
+# Input {#examples-engine-input}
+
+@brief Using the @ref input-plugin plugin.
+
+This example shows how the @ref scene-input plugin can be used to handle user input.
+
+The plugin function is included from the @ref engine/input/plugin.hpp header.
+
+@snippet input/main.cpp Adding the plugin
+
+The Input plugin requires a @ref cubos::engine::InputBindings "InputBindings" asset to be set.
+Let's take a look at the file the sample uses.
+
+@include assets/sample.bind
```

Comment:
This file is way too long to be included, I would remove some actions. I think we can have only the `next-showcase`, `x-or-z`, `shift-space`, `ctrl-shift-space` actions (see other review as why to remove last two actions). Please adapt the sample accordingly.